ramp-engine/ramp_engine/tests/test_remote_worker.py

```diff
@@ -0,0 +1,55 @@
+import os
+from pathlib import Path
+
+from ramp_engine.remote import _check_dask_workers_single_machine
+from ramp_engine.remote import _serialize_folder, _deserialize_folder
+import pytest
+
+
+def test_dask_workers_single_machine():
+    workers_nthreads = {
+        'tcp://127.0.0.1:38901': 4,
+        'tcp://127.0.0.1:43229': 4,
+        'tcp://127.0.0.1:46663': 4
+    }
+
+    assert _check_dask_workers_single_machine(workers_nthreads.keys()) is True
```

Comment:
I don't think you need the `is True`

Reply:
without it, any not None or not False object would pass the assert. Here I wanted to check that the output is explicitly `True`

Reply:
right